packages/core/src/divider/divider.element.scss

```diff
@@ -0,0 +1,37 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --color: #{$cds-token-color-neutral-300};
+  --size: #{$cds-token-space-size-1};
+  --padding: 0;
+
+  display: block;
+  width: 100%;
+  padding: var(--padding);
+}
+
+.private-host {
+  height: var(--size);
+  background: var(--color);
+}
+
+:host([orientation='vertical']) {
+  // align-self and the height allow the vertical divider to fill up its parent container inside a layout
+  // - height will likely have to be set via CSS if the vertical divider is used outside of a layout
+  // - if a container is set to a smaller height than its contents should allow the vertical divider
+  //   will creep up on the screen above the container.
+  align-self: stretch;
+  height: inherit;
+  width: var(--size);
+}
+
+:host([orientation='vertical-fill']) {
+  // for use outside of layouts. must be used inside a container with a set height.
+  height: 100%;
+  width: var(--size);
```

Comment:
In this demo I changed it to use `vertical` instead of `vertical-fill` and it seems to still work. Do we still need `vertical-fill`?

https://5f624cda9b99d300d803db2c--vmware-clarity.netlify.app/storybook/core/?path=/story/components-preview-divider-getting-started--page#vertical-fill-divider

Reply:
🤔

Reply:
I'll have to investigate that. It does seem to continue working. So the vertical-fill may no longer be needed.

Reply:
Removed.